pydantic/schema.py

```diff
@@ -87,6 +87,16 @@
 TypeModelSet = Set[TypeModelOrEnum]
 
 
+def _apply_modify_schema(modify_schema: Callable, field: ModelField, schema: Dict[str, Any]):
```

Comment:
The `field` should be `Optional[ModelField]` and not `ModelField`, since `__modify_schema__()` is not _only_ called for fields. If it's called for entire model, there is no `ModelField` in this context.

Reply:
Fixed in newer revision :point_up: